src/Sylius/Component/Core/Distributor/ProportionalIntegerDistributor.php

```diff
@@ -0,0 +1,38 @@
+<?php
+
+/*
+ * This file is part of the Sylius package.
+ *
+ * (c) Paweł Jędrzejewski
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Sylius\Component\Core\Distributor;
+
+/**
+ * @author Mateusz Zalewski <mateusz.zalewski@lakion.com>
+ */
+class ProportionalIntegerDistributor implements ProportionalIntegerDistributorInterface
```

Comment:
Maybe it would be worthy to close this implementation with `final` keyword? What could change here? Even if somebody would like to override this class, he can just create a new one.